File: C/10.28/add.c
//Paola Frunzio
//10.28.2020
//add two numbers with a function

#include <stdio.h>
#include <cs50.h>

int addNum();

int main(void)
{
    int sum;
    
    sum = addNum();
    
    printf("%i\n", sum);
}

int addNum()
{
    int num1, num2;
    num1 = get_int("Number: ");
    num2 = get_int("Number: ");
    return num1 + num2;
}

File: Python/1.15/add.py
# Paola Frunzio
# 1.15.2021
# input two number and get sum with a function

from cs50 import get_int

def getSum(num1, num2):
    return num1 + num2

# program
one = get_int("Enter a number: ")
two = get_int("Enter a number: ")

total = getSum(one, two)

print(f"The sum of {one} and {two} is {total}.")